llvm/test/Transforms/NewGVN/assume_dominating_icmp.ll

```diff
@@ -0,0 +1,62 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; RUN: opt -passes=newgvn -S < %s | FileCheck %s
+
+@c = global i32 0, align 4
+
+; Function Attrs: nounwind optsize uwtable
+define dso_local i32 @main(i1 %cond, i32 %0, i32 %1) {
```

Comment:
Can drop dso_local. I'd also recommend to avoid the use of unnamed values, i.e. replace `i32 %0` with `i32 %arg0` etc.

Reply:
I am sorry I missed this comment. I created a new pr: https://github.com/llvm/llvm-project/pull/66711